test/e2e/lib/framework/createTest.ts

```diff
@@ -20,7 +20,7 @@ const DEFAULT_RUM_CONFIGURATION = {
   trackLongTasks: true,
   telemetrySampleRate: 100,
   telemetryConfigurationSampleRate: 100,
-  enableExperimentalFeatures: [],
+  enableExperimentalFeatures: ['fetch_duration'],
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I'm not sure we want to enable it for all tests, as we want to ensure that the SDK works without this feature too.
* If you added a test that requires this flag to work, you can use `withRum({ enableExperimentalFeatures: ['fetch_duration'] })`
* if you want to make sure an existing test still works with this flag, you can duplicate it and use `withRum({ enableExperimentalFeatures: ['fetch_duration'] })` on one of them.